server/src/main/java/org/elasticsearch/search/aggregations/metrics/NumericMetricsAggregation.java

```diff
@@ -40,5 +40,12 @@
          */
         double value(String name);
 
+        /**
+         * Return the result of 1 value by name as string
+         *
+         * @param name of the value
+         * @return the value as string
+         */
+        String valueAsString(String name);
```

Comment:
`InternalNumericsMetricAggregation.MultiValue` already had this method, so all I needed is exposing it, however note that SingleValue calls it `getValueAsString`.

Let me know, if I should rename to `getValueAsString` to have a more consistent naming.